sigs/rules-authors/CHARTER.md

```diff
@@ -0,0 +1,71 @@
+# SIG Rules Authors
+
+## Objective
+
+Authors of commonly used Bazel rulesets can share technical approaches for solving common problems, speak with a coherent voice when interacting with the core Bazel team, and provide a more consistent experience for Bazel end-users.
+
+> The term "rulesets" is used to refer to all Bazel extensions, including Editor Plugins and Starlark shared libraries.
+
+## Participating
+
+### As a contributor
+
+Membership has been originally granted to maintainers of the "canonical" rulesets: those in the bazelbuild GitHub org, or listed in the [recommended rulesets](https://docs.bazel.build/versions/4.1.0/rules.html#recommended-rules).
+
+Criteria for new members is still under discussion: <https://github.com/bazel-contrib/SIG-rules-authors/issues/1>
+
+### As a company
+
+The SIG is funded by companies that rely on the community-maintained rulesets.
+We need your support to continue providing the software you depend on!
+
+> As of November 2021, Google is _not_ a funder of the SIG.
+
+Companies can contribute in several ways:
+
+- _Intellectual property_: Upstream fixes and features your organization has made. Donate proprietary rulesets developed in-house.
+- _Engineering time_: Give your enthusiastic developers some dedicated "20% time" to make targeted contributions that benefit your use cases.
+- _Financial support_: The SIG plans to accept monetary contribution, likely using <https://opencollective.com>.
+
+Participation benefits your company:
+
+- Recruit and retain talented engineers who want to work in open source.
+- Attribution of your contributions builds respect for your brand in the community.
+- Avoid merge conflicts when upstream changes break your private patches to Bazel rules.
+
+Please get in touch with us if you think your company may be interested. See the contact info below.
+
+## Resources
+
+Contact the SIG:
+
+- on Slack: `#rules` channel in https://slack.bazel.build
+- by e-mail: bazel-contrib@googlegroups.com
+- E-mail archives: https://groups.google.com/g/bazel-contrib/
+- If you need to reach out privately, email the SIG Leads listed below.
+
+[Meeting notes](https://docs.google.com/document/d/1YGCYAGLzTfqSOgRFVsB8hDz-kEoTgTEKKp9Jd07TJ5c/edit#)
+ from prior meetings and upcoming agenda.
+
+The GitHub repo containing governance and administration:
+<https://github.com/bazel-contrib/SIG-rules-authors>
+
+This SIG was originally proposed here: <https://github.com/bazelbuild/proposals/blob/main/designs/2021-08-10-rules-authors-sig.md>
+This document includes some potential activities of the SIG.
```

Comment:
Would you consider please to move the sample activities into the objectives section so that this charter stands on its own?

Reply:
actually the request-template had the prompt

>  *List potential goals for the group*
and the charter-template does not. So to follow the instructions closely, I'll inline the content into the PR description

Reply:
Very fair point. Let's leave it like this and not overthink it. Also as initial problems and other work items may or will shift in future, so the idea is probably that the charter (i.e. the mission) stays more constant.